### DISCUSSION TITLE
Support format on paste

### DISCUSSION THREAD

**Discussion by luabud** [microsoft/pylance-release#4342] [Enhancement]
When pasting Python code in the editor, it'd be nice if Pylance could automatically indent it properly. Similar to how it does auto indentation, but when pasting code.

**Comment by iosorin** [comment]
# Source - https://stackoverflow.com/a/68495092
# extension.paste-indent

[
    {
        "key": "ctrl+v",
        "command": "extension.paste-indent",
        "when": "editorTextFocus && editorLangId == 'python' && !editorReadonly"
    }
]
